### PR TITLE
Add igvm .signed and .evts.pb files

### PR DESCRIPTION
**Pull request by deeglaze**
More useful for packaging

### CODE REVIEW

File: endorse/commit.go [hunk 1]
```diff
@@ -267,10 +272,12 @@ func snapshotEndorsement(ctx context.Context, cops ChangeOps, endorsement *epb.V
...
 	fwPath := ec.VCS.ReleasePath(ctx, path.Join(ec.SnapshotDir, ec.ImageName))
+	fwSigPath := fmt.Sprintf("%s.signed", fwPath)
+	fwEvtsPath := fmt.Sprintf("%s.evts.pb", fwPath)
 	svsmPath := ec.VCS.ReleasePath(ctx, path.Join(ec.SnapshotDir, "svsm.igvm"))
-	sigPath := fmt.Sprintf("%s.signed", fwPath)
-	evtsPath := fmt.Sprintf("%s.evts.pb", fwPath)
-	if err := writeEndorsement(ctx, sigPath, endorsement, cops); err != nil {
+	svsmSigPath := fmt.Sprintf("%s.signed", svsmPath)
+	svsmEvtsPath := fmt.Sprintf("%s.evts.pb", svsmPath)
+	if err := writeEndorsement(ctx, []string{fwSigPath, svsmSigPath}, endorsement, cops); err != nil {
```
Comment (AdamCDunlap) [inline endorse/commit.go:280]: Do we only want to write the svsm.igvm.signed and svsm.igvm.events.pb file if SvsmImage is set?

File: endorse/commit.go [hunk 2]
```diff
@@ -284,25 +291,22 @@ func snapshotEndorsement(ctx context.Context, cops ChangeOps, endorsement *epb.V
...
-	if err := cops.SetBinaryWritable(ctx, fwPath); err != nil {
-		return fmt.Errorf("could not set %q type to binary: %w", fwPath, err)
-	}
-	if err := cops.SetBinaryWritable(ctx, evtsPath); err != nil {
-		return fmt.Errorf("could not set %q type to binary: %w", evtsPath, err)
-	}
-	if len(ec.SvsmImage) != 0 {
-		if err := cops.SetBinaryWritable(ctx, svsmPath); err != nil {
-			return fmt.Errorf("could not set %q type to binary: %w", svsmPath, err)
+	for _, path := range binPaths {
```
Comment (AdamCDunlap) [inline endorse/commit.go:307]: Can't you reuse the files slice instead of making binPaths as well?

Comment (deeglaze) [reply endorse/commit.go]: Done.

File: endorse/commit.go (hunk 1 above)
Comment (deeglaze) [reply endorse/commit.go]: Done.

**Review by AdamCDunlap — APPROVED** [review]

**Pull request merged**